docs/src/quickstart/index.md

```diff
@@ -1,35 +1,48 @@
-> - Have installed the [`forc index`](http://localhost:3000/plugins/forc-index.html) plugin via [`fuelup`](http://localhost:3000/getting-started/application-dependencies/fuelup.html)
+1. How to bootstrap an environment for use with a Fuel indexer.
+2. How to use the `forc index` plugin to create, build, and deploy indicies.
+3. How to query our newly created indices for data using GraphQL.
+
+> IMPORTANT: Docker is a prerequisite for using this Quickstart. If Docker is not installed on your machine, please review the Docker installation instructions [here](https://docs.docker.com/engine/install/).
+
+## 1. Setting up your environment
 
-## What we'll do
+As a user of the Fuel indexer, you shouldn't require _too many_ dependencies. The indexer binaries available via `fuelup` are precompiled with all required dependencies, and optimized for size.
 
-In this Quickstart we're going to write a simple Sway 🌴 smart contract, then
-build and deploy and index that saves events from this contract into a database.
+In this Quickstart, we'll only be using Postgres. Further, instead of going through the
+drudgery of installing Postgres on your local machine, we'll just use [the `latest`
+Postgres docker image](https://hub.docker.com/_/postgres) via the following command: `docker pull postgres`.
 
-- [Writing a contract](#writing-a-contract)
-- [Create and deploy an index](#create-and-deploy-an-index)
-  - [Initializing an index](#1-initialize-a-new-index-project)
-  - [GraphQL data models](#2-graphql-data-models)
-  - [Update index manifest](#3-update-index-manifest)
-  - [Write your index code](#4-write-your-index-code)
-  - [Compile the index](#5-compile-the-index)
-  - [Start the index service & deploy your index](#6-start-the-service--deploy-your-index)
+## 2. Using the `forc-index` plugin
 
-## What you'll need
+- The primary means of interfacing with the Fuel indexer **for index development** is the [`forc-index` CLI tool](https://crates.io/crates/forc-index).
+- `forc-index` is a [`forc`](https://github.com/FuelLabs/sway/tree/master/forc) plugin specifically for the Fuel indexer service.
+- For convenience, the `forc-index` binary is already included with `fuelup`.
+  - [`fuelup` setup instructions](https://github.com/FuelLabs/fuelup)
 
-For this Quickstart we'll need a few components that include Docker, Postgres,
-and the Fuel Indexer service. In order to see which components you have
-installed already, simply use the `forc index check` command, which will check
-for these executables in your `$PATH`.
+> IMPORTANT: `fuelup` also contains the `fuel-indexer` binary. This is the binary that runs the Fuel indexer service.
+
+If your `forc-index` and `fuel-indexer` binaries were successfully installed via `fuelup`, your `PATH` should contain:
+
+```text
+➜  which forc-index
+
+/Users/me/.fuelup/bin/forc-index
+
+➜  which fuel-indexer
+
+/Users/me/.fuelup/bin/fuel-indexer
```

Comment:
@hrryndrsn 
- A bit confused by this
- What is `forc-indexer`? Did you mean `fuel-indexer`?
- Latest commit should add a brief `fuel-indexer` explanation.

Reply:
the naming convention i dont really understand personally

Reply:
The `fuel-indexer` binary predates the `forc-index` binary. Additionally, the `forc-index` binary is provided for developer experience for users who are also writing Sway and interacting with the rest of the Fuel ecosystem; it's not required to be able to run and/or interact with the Fuel indexer itself. One can run the compiled binaries for the Fuel indexer (and its API server) and never interact with `forc` at all.

That being said, we recommend using `forc index` as the primary way of interacting with the Fuel indexer because it leads to less friction and mental load for users; instead of having to determine what flags are needed to run the service or compile the WASM modules, we make an informed assumption and provide an easy-to-use method for the majority of users.